src/fusion/fusion_pipeline.py

```diff
@@ -79,10 +79,10 @@ def process_file(self, in_file, out_file=None, mask=None):
 
         tilecode = get_tilecode_from_filename(in_file)
         pointcloud = read_las(in_file)
-        points = {'x': pointcloud.x, 'y': pointcloud.y, 'z': pointcloud.z}
+        points = np.vstack((pointcloud.x, pointcloud.y, pointcloud.z)).T
 
         if mask is None:
-            mask = np.full((len(points['x']),), True)
+            mask = np.ones((len(points),), dtype=bool)
```

Comment:
Deze mask meegeven aan process_cloud. Hoef je hem daar niet meer aan te maken

Reply:
Doe ik ook toch? :D

Reply:
Nee Line 48 kan nooit none zijn

Reply:
In principe wel, als process_cloud() vanuit ergens anders aangeroepen zou worden. Of we moeten ervoor kiezen dat dat niet mag.

Reply:
Ah ja ik ging ervan uit dat dat niet kon